src/tests/ftest/server/multiengine_persocket.py

```diff
@@ -231,9 +247,9 @@ def test_multiengines_per_socket(self):
             "sudo cp /etc/daos/daos_server.yml_4 /etc/daos/daos_server.yml",
             "sudo systemctl start daos_server.service"]
         for cmd in start_server_cmds:
-            results = run_pcmd(self.hostlist_servers, cmd, timeout=90)
+            results = run_remote(self.log, self.hostlist_servers, cmd, timeout=90)
         # Check for server start status
-        if results[0]['exit_status']:
+        if not results.passed:
```

Comment:
shouldn't this be indented to be checking each of the commands that are iterated through in the above for block succeeded?

Reply:
Yes it should
```suggestion
            if not results.passed:
```